geode-core/src/integrationTest/resources/org/apache/geode/distributed/LocatorLauncherRemoteWithCustomLoggingIntegrationTest_log4j2.xml

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Licensed to the Apache Software Foundation (ASF) under one or more
-  ~ contributor license agreements.  See the NOTICE file distributed with
-  ~ this work for additional information regarding copyright ownership.
-  ~ The ASF licenses this file to You under the Apache License, Version 2.0
-  ~ (the "License"); you may not use this file except in compliance with
-  ~ the License.  You may obtain a copy of the License at
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more contributor license
```

Comment:
These header changes makes me think that maybe someone's IntelliJ formatting is setup differently?  Do we want these changes?

Reply:
Others may not have IntelliJ copyright configured. I added directions to the Apache Geode README to make it identical to my setup but someone may have deleted it. My guess is that Jake deleted too much. But yes, I wanted these changes. I don't want to have to make the .xml files use a different config than the .java files and this is how it formats it.